can_tools/scrapers/official/NC/nc_demographics.py

```diff
@@ -0,0 +1,35 @@
+from tableauscraper import TableauScraper
+
+from can_tools.scrapers import variables
+from can_tools.scrapers import NCVaccine
+
+# This uses the tableauscraper module from https://github.com/bertrandmartel/tableau-scraping
+# which "mimics" the actions on a dashboard (like filtering from dropdowns, clicking on maps etc.)
+# The docs are pretty good and there are helpful examples there, but knowing what filters/parameters to
+# use when and in what order can be unclear.
+
+# In general, I usually go to the dashboard with the network inspector open,
+# navigate to the data we want to pull, and try to match the names of the calls to the TableauScraper filtering functions.
+# E.g. a network call that sends a `set-parameter-value` corresponds to the workbook.GetParameter(...) method,
+# and a `select` call corresponds to the `worksheet.select(...)` function.
+
+
+class NCVaccineAge(NCVaccine):
+    # map wide form column names into CMUs
+    variables = {
+        "": variables.PERCENTAGE_PEOPLE_INITIATING_VACCINE,
+        "": variables.PERCENTAGE_PEOPLE_COMPLETING_VACCINE,
+    }
+
+    def fetch(self):
+        engine = TableauScraper()
+        engine.loads(
+            "https://public.tableau.com/views/NCDHHS_COVID-19_Dashboard_Vaccinations/VaccinationDashboard"
+        )
+        engine = engine.getWorkbook()
+        sp = engine.goToStoryPoint("2")
+        wb = sp.setParameter("County", "Wake County")
+
+        # i feel like with the set paramater that this should be returning data in the Race_county... tables
+        # but nothing :/
```

Comment:
@smcclure17 Not sure if it helps but while I don't see anything under "Race_county", I do see data under "Race_Percent_county"...  But this is % as in "The percent of the black population that is fully vaccinated" which I'm not sure we can / want to represent?

Anyway, not sure if this is helpful, but thought I'd mention it in case it is.

Reply:
Yeah, I'm not sure that's useful on its own, (without population data that we can multiply by), but it is promising that there's some sort of county-level data there. Seems like if that data is here then the rest must be somewhere, but not sure where :/

I think I'm going to table this for now if that's ok, and try to come back to it whenever I find some free time.

Thanks for taking a look into it! 😃

Reply:
FYI- I poked at this a tiny bit more and I think it *might* have raw counts too, but I'm not 100% sure if that's what these numbers are or not.  We'd need to sanity-check.

![image](https://user-images.githubusercontent.com/206364/141827182-b8a0dc93-4bc6-48e2-b482-5f6745a1860e.png)

In any case, I'm 100% okay with tabling this.